src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java

```diff
@@ -9,6 +10,7 @@
 public enum TFM_PlayerRank
 {
     DEVELOPER("a " + ChatColor.DARK_PURPLE + "Developer", ChatColor.DARK_PURPLE + "[Dev]"),
+    ADMIN("a " + ChatColor.RED + "Administrator", ChatColor.RED + "[Administrator]"),
```

Comment:
Change this to:
Executive("a " + ChatColor.RED + "Executive", ChatColor.RED + "[Executive]"),